Functions/xASL_bids_CreateDatasetDescriptionTemplate.m

```diff
@@ -0,0 +1,105 @@
+function [json] = xASL_bids_CreateDatasetDescriptionTemplate(varargin)
```

Comment:
Contemplating what is the best input type... 
Mostly, we will not be filling the varargin ourself, but the varargin values will be given to us in a JSON already? 

Basically, we will receive an incomplete dataset with a JSON containing dataset_description fields among other fields. And our task will be to filter out the useful information.

So my guess is that we will call this function and give it a JSON-struct on the entry and it will filter out the good parameters, check them and provide the missing ones. Restructuring the JSON-input into separate input variables would be an extra work then.

Do you agree, or you had a different use case in your mind?

Reply:
Ahhh, now I understand what you want to use it for 👍 My use case here was a bit more simple so I used some copy paste from the Initialize script I'm working on. But I think I understand what you have in mind and I can implement it.

Reply:
Check out bcfe3e6.